<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-65O6UO6T" failures="0" tests="1" name="Default test" time="0.037" errors="0" timestamp="2024-04-11T22:56:23 IST">
  <testcase classname="com.ShopperStackGUtility.BaseTest" name="@BeforeTest bt" time="0.0">
    <failure type="java.lang.Error" message="Unresolved compilation problems: 
	Syntax error, insert &amp;quot;VariableDeclarators&amp;quot; to complete LocalVariableDeclaration
	Syntax error, insert &amp;quot;;&amp;quot; to complete LocalVariableDeclarationStatement
	report cannot be resolved
	report cannot be resolved
	report cannot be resolved
">
      <![CDATA[java.lang.Error: Unresolved compilation problems: 
Syntax error, insert "VariableDeclarators" to complete LocalVariableDeclaration
Syntax error, insert ";" to complete LocalVariableDeclarationStatement
report cannot be resolved
report cannot be resolved
report cannot be resolved

at com.ShopperStackGUtility.BaseTest.bt(BaseTest.java:46)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:69)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:393)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:326)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:661)
at org.testng.TestRunner.beforeRun(TestRunner.java:650)
at org.testng.TestRunner.run(TestRunner.java:612)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1228)
at org.testng.TestNG.runSuites(TestNG.java:1134)
at org.testng.TestNG.run(TestNG.java:1101)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @BeforeTest bt -->
  <testcase classname="com.ShopperStackGUtility.BaseTest" name="@BeforeClass bc" time="0.0">
    <skipped/>
  </testcase> <!-- @BeforeClass bc -->
  <testcase classname="com.ShopperStackGUtility.BaseTest" name="@BeforeMethod bm" time="0.0">
    <skipped/>
  </testcase> <!-- @BeforeMethod bm -->
  <testcase classname="com.ShopperStackGUtility.BaseTest" name="@AfterMethod am" time="0.0">
    <skipped/>
  </testcase> <!-- @AfterMethod am -->
  <testcase classname="com.ShopperStackGUtility.BaseTest" name="@AfterClass ac" time="0.0">
    <skipped/>
  </testcase> <!-- @AfterClass ac -->
  <testcase classname="com.ShopperStack.testScript.Verify_user_is_able_add_address" name="addAddress" time="0.0">
    <skipped/>
  </testcase> <!-- addAddress -->
</testsuite> <!-- Default test -->
